src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -206,7 +206,7 @@ <h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} {{ entryTypeT
     </fieldset>
   </div>
   <div mat-dialog-actions fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="10px">
-    <button type="button" mat-flat-button class="secondary-1" mat-dialog-close>Cancel</button>
+    <button type="button" mat-flat-button class="secondary-1" mat-dialog-close>OK</button>
```

Comment:
Hmmm, not sure this is doing quite what was described. 

Basically, we want either
1) "Cancel" and "Save Changes" button or 
2) "OK" when "Save Changes" is not present

I think this changes it to OK across the board, so you could end up with "OK" and "Save Changes"

Reply:
Oh I think I misunderstood the ticket then, I thought it just wanted OK, will change